packages/core/data-transfer/package.json

```diff
@@ -26,19 +26,19 @@
   ],
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "files": [
+    "./dist/**/!(*.d.ts.map)"
```

Comment:
no need to show off

Reply:
for some reason my editor decided to reorder the package.json files